bson/codec_options.py

```diff
@@ -37,7 +37,6 @@
     UuidRepresentation,
 )
 
-# Import RawBSONDocument for type-checking only to avoid circular dependency.
```

Comment:
This seems like a helpful comment. Why remove it and now the code below?

Reply:
Ah, I had removed the import itself at one point, I'll put the comment back